.github/workflows/tests-sandbox.yml

```
@@ -9,7 +9,7 @@ jobs:
     strategy:
       matrix:
         platform: [ubuntu-latest, macos-latest]
-        node-version: [14, 16]
+        node-version: [14, 16, 18, 20]
```

Hey @no2chem! This should resolve the conflicts and we can merge the PR.

```suggestion
        node-version: [16, 18, 20]
```